packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```diff
@@ -173,9 +173,646 @@
+              },
+              {
+                "$ref": "#/components/schemas/DetachRoleRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetRolePermissionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GrantPermissionRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/RevokePermissionRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetTransactionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetPendingTransactionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountTransactionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountAssetTransactionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetBlockRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/CallEngineRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetEngineReceiptsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/AddPeerRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/RemovePeerRequestParameters"
+              }
+            ]
+          }
+        }
+      },
+      "CreateAccountRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Create Account.",
+        "type": "object",
+        "required": [
+          "accountName",
+          "domainId",
+          "publicKey"
+        ],
+        "properties": {
+          "accountName": {
+            "type": "string"
+          },
+          "domainId": {
+            "type": "string"
+          },
+          "publicKey": {
+            "type": "string"
+          }
+        }
+      },
+      "SetAccountDetailRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Set Account Detail.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "key",
+          "value"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "key": {
+            "type": "string"
+          },
+          "value": {
+            "type": "string"
+          }
+        }
+      },
+      "CompareAndSetAccountDetailRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Compare And Set Account Detail.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "key",
+          "value",
+          "oldValue"
```

Comment:
This is optional, not required:
https://iroha.readthedocs.io/en/develop/develop/api/commands.html#compare-and-set-account-detail